portfolio/src/main/webapp/script.js

```diff
@@ -45,26 +45,60 @@ function openConnectModal() {
   modal.style.display = "block";
 }
 
-function closeIntroModal(){
+function openQuestionModal() {
+  const modal = document.getElementById("myQuestionModal");
+  modal.style.display = "block";
+}
+
+function openViewModal() {
+  const modal = document.getElementById("myQuestionViewModal");
+  modal.style.display = "block";
+  fetch('/data').then(response => response.json()).then((Questions) => {
+  const questionContainer = document.getElementById("question-container");
+  questionContainer.innerHTML = '';
+  for (let i = 0; i < Questions.length; i++) {
+    const createElement = document.createElement("li");
+    var createNode = document.createTextNode(Questions[i].name + " asked: " + Questions[i].text);
+    createElement.appendChild(createNode);
+    questionContainer.appendChild(createElement);
+    if (i === 9) break;}
+  });    
```

Comment:
Your indentation for code inside fetch('/data') is off (lines 57-64). Indent everything between there over 2 spaces.